src/main/java/g2601_2700/s2699_modify_graph_edge_weights/Solution.java

```diff
@@ -0,0 +1,82 @@
+package g2601_2700.s2699_modify_graph_edge_weights;
+
+// #Hard #Heap_Priority_Queue #Graph #Shortest_Path
+// #2023_09_14_Time_88_ms_(85.25%)_Space_49.9_MB_(85.25%)
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.PriorityQueue;
+
+@SuppressWarnings("java:S135")
+public class Solution {
+    public int[][] modifiedGraphEdges(
+            int n, int[][] edges, int source, int destination, int target) {
```

Comment:
Please fix sonar warning

A "Brain Method" was detected. Refactor it to reduce at least one of the following metrics: LOC from 68 to 64, Complexity from 20 to 14, Nesting Level from 3 to 2, Number of Variables from 19 to 6.